.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    },
+    extends: 'eslint:recommended',
```

Comment:
Out of curiosity, any experience with `eslint --fix`? It seems like a tool that (with care) could make minor style compliance tractable.

http://eslint.org/docs/user-guide/command-line-interface